### ISSUE TITLE
Controller

### ISSUE THREAD

**Issue by ndac-todoroki** [Phi-phi/oop-server#5]
「こういうの作ります！」
　　↓
実装・プルリク
　　↓
もう片方が確認、コメント
　　↓
マージ

な感じでいいと思うのでとりあえず同じ作業をしないようにここで宣言する感じがいいかなと

**Comment by Phi-phi** [comment]
はい。
とりま通信関係は私の方でやろうと思うんだけどいいだろうか？
アプリからデータ受け取るのと、json渡すあたりかな。
コントローラーわけてやるかな？

**Comment by ndac-todoroki** [comment]
うん。まずは何かしらのString受け取ったらなにかしらのjson返せば向こうの動作テスト的にはいいと思うしDB周りへの働きかけは後ででもいいのでよろしく
その2つでコントローラー自体を分けるってこと？うーんメソッド分けるだけでいい気もするけど…でもとりあえずは考えた感じで作ってみていいと思う！
雛形でプルリクしてくれるとわかりやすいかも

**Comment by Phi-phi** [comment]
phi/devブランチ作成、自分のコミットはここにプッシュします。

**Comment by Phi-phi** [comment]
大丈夫そうなら、masterにマージしようと思います。
